docs/structured-streaming-kafka-integration.md

```diff
@@ -310,6 +310,27 @@ The following configurations are optional:
 
 <table class="table">
 <tr><th>Option</th><th>value</th><th>default</th><th>query type</th><th>meaning</th></tr>
+<tr>
+  <td>startingOffsetsByTimestamp</td>
+  <td>json string
+  """ {"topicA":1000,"topicB":2000} """
+  </td>
+  <td>none (the value of `startingOffsets` will apply)</td>
```

Comment:
As a general comment all other places in the doc double quote used instead of single.

Reply:
After the last commit I still see several single quotes. Is this change not added?

Reply:
Actually it is already not consistent where we should use \`, or `'`, or `"`. And the reason of using \` is to highlight which we can't get via using `"`.

Reply:
If you mean highlight just like in github comments then I can say it's not working in the generated html document.

Reply:
Yeah Spark doc renderer doesn't look like highlighting it, though it still show it slightly differently hence \` is still heavily used in doc.

https://spark.apache.org/docs/latest/streaming-custom-receivers.html

onStart(), onStop(), store(data), etc are wrapped with \`, and they are rendered a bit differently.

Reply:
I'm accepting the highlight way but doesn't work when I've generated the html. I've double checked the page what you've linked and there it looks good.

![Screen Shot 2019-03-20 at 10 25 46 AM](https://user-images.githubusercontent.com/18561820/54673635-ec678f00-4afa-11e9-91a1-dd750e058f4d.png)

Reply:
Hmm... interesting. I'm guessing it doesn't render \` within table related tag, but not sure. I'll just wrap them with `<code>...</core>` which should work.

Reply:
Addressed. Could you rerun the render and see whether it addresses your review comment?

Reply:
Now it's fine.